big_tests/tests/connect_SUITE.erl

```diff
@@ -303,6 +303,7 @@ reset_stream_noproc(Config) ->
     [RcvPid] = children_specs_to_pids(rpc(mim(), supervisor, which_children, [ejabberd_receiver_sup])),
     MonRef = erlang:monitor(process, C2sPid),
     ok = rpc(mim(), sys, suspend, [C2sPid]),
+    timer:sleep(100),
```

Comment:
Could we use for example, `erlang:process_info` to see if the process is in `suspend` state and only then proceed? Today `100ms` may be enough, the really thing we are wait for is that the process is in `suspend` state as far as I understand.

Reply:
You may want to reuse `mongoose_helper/wait_until/3` function to retry the check if needed.

Reply:
sounds like good idea.